src/squlearn/util/decompose_to_std.py

```diff
@@ -0,0 +1,37 @@
+from qiskit.circuit.library import standard_gates
+from qiskit.circuit import QuantumCircuit
+
+
+def decompose_to_std(circuit: QuantumCircuit, gate_list: list = []) -> QuantumCircuit:
```

Comment:
gate_list: Union[None,list] = None would be better to signal it is a optional input

Reply:
That's what I meant, isn't it?

Reply:
sorry, missed the = None part 🤦‍♂️